### PR TITLE
Implement 404 not found page #281

### PR DESCRIPTION
**Pull request by hoangvu96z**

### CODE REVIEW

File: packages/app-lixi/src/pages/404.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                .message {
+                    font-weight: normal
+                }
+            }
+        }
+        `;
+	return <>
+		<FourOhFourPage>
+			<div className='container'>
+				<img src="/images/lotus-logo-small.png" alt="lotus" />
```
Comment (vince8x) [inline packages/app-lixi/src/pages/404.tsx:32]: It seems the new image in the figma design is not included?

File: packages/app-lixi/src/pages/404.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import Link from 'next/link';
```
Comment (vince8x) [inline packages/app-lixi/src/pages/404.tsx:1]: The file should be ident by 2 spaces instead of 4

**Comment by hoangvu96z** [comment]
image

File: packages/app-lixi/src/components/Layout/MainLayout/index.tsx [hunk 3]
```diff
@@ -211,7 +212,7 @@ const MainLayout: React.FC = (props: MainLayoutProps) => {
                 <ModalManager />
                 <>
                   <AppContainer>
-                    <Layout>
+                    {!isNotFoundPage ? <Layout>
```
Comment (vince8x) [inline packages/app-lixi/src/components/Layout/MainLayout/index.tsx:215]: I think we should create new layout ưith no sidebar for this kind of page. Maybe "BareLayout".
Example:
```
ClaimPage.Layout = ({ children }) => <ClaimedLayout children={children} />;
```

File: packages/app-lixi/src/pages/404.tsx [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+  .main-section-layout {
+    @media (max-width: 768px) {
+      padding-right: 0 !important;
+    }
+  }
+`;
+
+type EmptyLayoutProps = React.PropsWithChildren<{}>;
+
+const EmptyLayout: React.FC = (props: EmptyLayoutProps) => {
```
Comment (vince8x) [inline packages/app-lixi/src/pages/404.tsx:100]: Could we move the empty layout to the Layout folder?

File: packages/app-lixi/src/components/Layout/EmptyLayout/index.tsx [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+  .main-section-layout {
+    @media (max-width: 768px) {
+      padding-right: 0 !important;
+    }
+  }
+`;
+
+type EmptyLayoutProps = React.PropsWithChildren<{}>;
+
+const EmptyLayout: React.FC = (props: EmptyLayoutProps) => {
```
Comment (vince8x) [inline packages/app-lixi/src/components/Layout/EmptyLayout/index.tsx:101]: This file seems to use 4 spaces indentation. Could you please run yarn precommit to lint the file to 2 spaces?

**Pull request merged**